src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java

```diff
@@ -49,12 +49,15 @@ public void testGetMicrosoftTimeZoneNameBad() {
     // null-argument is not allowed.
     try {
       Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));
-    } catch (final IllegalArgumentException ignored) {}
-
-    // Case-insensitive ID is not supported.
-    checkGetMicrosoftTimeZoneName("africa/abidjan", "UTC");
+    } catch (final IllegalArgumentException ignored) {
+    }
   }
 
+  @Test
+  public void testCaseInsensitiveIdNotSupported() {
+    // Case-insensitive ID is not supported so the map look up returns null.
+    checkGetMicrosoftTimeZoneName("africa/abidjan", null);
```

Comment:
To be honest, this isn't much of a test. All it does is validate that the string can't be found in ```TimeZoneUtils.createOlsonTimeZoneToMsMap()```. It should return ```null``` since it's not in the map, but don't have a good explaination for why that is not that case on the build machine, and on your local system.

Unfortunately I can't reproduce the failure so I guess I'm going to withdraw the PR. It's strange to me that #392 is also"fixing" an issue that can't be reproduced by the build. It's strange.